demisto_sdk/commands/lint/tests/test_pylint_plugin/xsoar_level_checker_test.py

```diff
@@ -164,6 +164,51 @@ def main() -> bool:
         with self.assertNoMessages():
             self.checker.visit_functiondef(node)
 
+    def test_not_implemented_error_exists_in_if_clause(self):
+        """
+        Given:
+            - String of a code part which is being examined by pylint plugin.
+        When:
+            - Main function that in the if clause raises a NotImplementedError.
+        Then:
+            - Ensure that the there was not message added to the checker.
+        """
+        node = astroid.extract_node("""
+            def main() -> bool:
+                try:
+                    if True:
+                        return True
+                    if command not in commands:
+                        raise NotImplementedError("this command was not implemented")
+                    else:
+                        return True
+                except Exception:
+                    pass
```

Comment:
Let's check elif too

```suggestion
                try:
                    if True:
                        return True
                    elif False:
                        return False
                    if command not in commands:
                        raise NotImplementedError("this command was not implemented")
                    else:
                        return True
                except Exception:
                    pass
```